src/components/Note/Note.tsx

```diff
@@ -1,33 +1,73 @@
-import './Note.scss';
-import avatar from 'assets/avatar.png';
-import { useSelector } from 'react-redux';
+import "./Note.scss";
```

Comment:
Maybe you could also set `cursor: pointer` for `.note` to visualize that it's clickable